### PR TITLE
Create missingNumberInArray.java

### PR DESCRIPTION
**Pull request by SHASHANK0712**
Hello Sir, I have written code for finding the only number in the range that is missing from the array. This was recently asked to me during an interview so I thought I should commit to this repository.

Kindly accept the pull request with hacktoberfest-accepted label.

### CODE REVIEW

File: bitManipulation/missingNumberInArray.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package bitManipulation;
```
Comment (Jiganesh) [inline bitManipulation/missingNumberInArray.java:1]: // https://leetcode.com/problems/missing-number/

package bitManipulation;

**Review by Jiganesh — CHANGES_REQUESTED** [review]

File: bitManipulation/missingNumberInArray.java (hunk 1 above)
Comment (SHASHANK0712) [reply bitManipulation/missingNumberInArray.java:1]: Okk will do them. Also can you tell if this repo has been excluded from Hacktoberfest 2022?

Comment (SHASHANK0712) [reply bitManipulation/missingNumberInArray.java:2]: I have done the changes

**Review by Jiganesh — APPROVED** [review]

**Comment by Jiganesh** [comment]
Does It show Excluded after adding Label ?

**Comment by SHASHANK0712** [comment]
Yes. It is showing Excluded now also.

**Pull request merged**